docs/core/configuration.md

```diff
@@ -634,12 +634,12 @@ What happens in reality is, however, a little bit different:
 #4 block - Alice
 ```
 
-That's because Bob is too fast at proposing blocks. This leaves Alice very
+That's because Bob is too fast at committing blocks. This leaves Alice very
 little chances to propose a block and not always be catching up. Note every
 block Bob creates needs a vote from Alice to constitute 2/3+.
 
 Imagine now there are ten geographically distributed validators. One of them
-(Bob) sets `timeout_propose` to 1s. Others have it set to 3s. Now, Bob won't be
+(Bob) sets `timeout_commit` to 1s. Others have it set to 3s. Now, Bob won't be
```

Comment:
This is not precise as well. If Bob is the proposer, it will receive its own proposal relatively fast and move to the `prevote` round step. At this point it will wait for `2/3+` Prevotes. If they match, Bob precommits the matching value (a block ID or nil). Otherwise, it schedules `timeout_prevote`, which has default duration of 1s.

This is to say that Bob will block until it gets `2/3+` Prevotes, and doesn't matter that it entered the `propose` step before other nodes (due to smaller `timeout_commit`).